### PR TITLE
spec using spec up

### PR DESCRIPTION
**Pull request by thereisnogabe**

### CODE REVIEW

File: spec/spec.md [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+**Participate**:
+~ [GitHub repo](https://github.com/TBD54566975/dap)
+~ [File a bug](https://github.com/TBD54566975/dap/issues)
+~ [Commit history](https://github.com/TBD54566975/dap/commits/main)
+
+## Abstract
+
+Decentralized Agnostic Paytags (DAPs) provide a standardized means to express human-friendly, resolvable payment handles that abstract over various currencies and payment networks. DAPs allow individuals to register memorable handles that resolve to a [[ref:Decentralized Identifier]] containing one or more associated payment addresses. This enables easy cross-app and cross-currency payments using a simple identifier. The syntax and data model used by DAPs is conformant to the [[spec:DID-CORE]] specification.
+
+<img src="resources/braindump.png" style="display: block; margin: 0 auto; padding: 1em 0; width: 100%; max-width: 620px;" />
```
Comment (thereisnogabe) [inline spec/spec.md:27]: not sure this will work

File: spec/spec.md [hunk 2]
```diff
@@ -0,0 +1,439 @@
...
+
+### Decentralized Agnostic Paytags
+
+To address these challenges, we propose a solution that enables seamless and user-friendly money transfers across different apps and currencies. The objective of this specification is to introduce a standardized approach called [[ref:Decentralized Agnostic Paytags]] (DAPs). A DAP is a memorable handle, similar to an email address (e.g., `handle@domain`), that allows individuals to send and receive money regardless of the app or currency being used.
+
+For example, consider a scenario where _Alice_ wants to pay back _Bob_ for a coffee:
+
+```txt
```
Comment (thereisnogabe) [inline spec/spec.md:47]: couldnt figure out how to format this

File: spec/spec.md [hunk 3]
```diff
@@ -0,0 +1,439 @@
...
+Alice: Yo Bobby boy! Thanks for the coffee. What's your DAP so I can pay you back?
+
+Bob: Anytime. DAP me up at waterboy@cash.app
+```
+
+With DAPs, _Alice_ can simply open her preferred payment app, enter _Bob's_ [[ref:DAP]] (`waterboy@cash.app`), and send the money. _Bob_ can then receive the money in his preferred app, regardless of whether it's the same as Alice's. This enables seamless transactions across various platforms, such as:
+
+- CashApp ⟷ YellowPay
```
Comment (thereisnogabe) [inline spec/spec.md:55]: need to add links...later

**Comment by thereisnogabe** [comment]
bump @mistermoe would be great to get this in before the new issues are addressed

**Comment by mistermoe** [comment]
thanks @decentralgabe ! will try to keep this up to date. though every time i look at spec up i feel like re-writing it before actually doing what i'm supposed to.

**Review by mistermoe — APPROVED** [review]

**Pull request merged**